corehq/apps/performance_sms/forms.py

```diff
@@ -106,3 +95,80 @@ def source_id(self):
         if self.config.template_variables:
             return self.config.template_variables[0].source_id
         return ''
+
```

Comment:
I think we can drop them for now. I do want to combine these two forms into a single one, but I wanted to get something functional that supported multiple sources out the door first